cmd/thor/main.go

```diff
@@ -80,6 +79,7 @@ func main() {
 			apiCallGasLimitFlag,
 			apiBacktraceLimitFlag,
 			apiAllowCustomTracerFlag,
+			apiEnableDeprecatedFlag,
```

Comment:
wonder if we should keep some name convention here:

- apiCallGasLimitFlag
- apiBacktraceLimitFlag
- apiAllowCustomTracerFlag
- apiEnableDeprecatedFlag
- enableAPILogsFlag
- apiLogsLimitFlag

so should we also change enableAPILogsFlag to apiEnableLogsFlag ?

Reply:
Makes sense, but I wouldn't block this PR for that change. Plus, naming has a way of opening up can of worms 😅